global_segment_map_node/cfg/generic.yaml

```diff
@@ -39,3 +42,10 @@ sliding_window:
 
 mesh_config:
   min_weight: 3
+
+icp:
+  enable_icp: false
+  keep_track_of_icp_correction: true
```

Comment:
Maybe this should be also set to false by default then?

Reply:
I would keep it true because if you enable icp, this you usually want to be true